### PR TITLE
refs #2903 remove <run-time-loaded: label> from Program::parseXXX(), …

### PR DESCRIPTION
**Pull request by tmandys**
…format_label is obsolete
**MAYBE WE WON'T NEED IT. HOLD ON.**

### CODE REVIEW

File: lib/QC_Program.qpp [hunk 1]
```diff
@@ -740,7 +740,7 @@ Program::destructor() {
     @param warning_mask An optional warning mask; see @ref warning_constants for values to combine by binary-or; if this argument is 0 or not given then no warnings will be checked or issued and the return value will always be @ref nothing
     @param source An optional source file name for the code being parsed; this is useful if sections of a file are parsed
     @param offset An optional line offset for use with the \a source parameter; this gives the line offset in the file to the code being parsed
-    @param format_label If this argument is @ref Qore::True "True", then the label is formatted as \c "<run-time-loaded: label>"; if @ref Qore::False "False", then it is passed as-is
+    @param format_label obsolete since %Qore 0.8.13
```
Comment (davidnich) [inline lib/QC_Program.qpp]: needs to read `obsolete / ignored since %Qore 0.8.13.7`

File: lib/QC_Program.qpp [hunk 2]
```diff
@@ -762,22 +762,21 @@ while (wh) {
 
     @throw PROGRAM-PARSE-ERROR this exception is thrown if any parse actions are started while the Program object has running threads
 
-    @since %Qore 0.8.7 the \a source, \a offset, and \a format_label arguments were added
+    @since %Qore 0.8.7 the \a source, \a offset, and \a format_label arguments were added. %Qore 0.8.13 the \a format_label is obsolete
```
Comment (davidnich) [inline lib/QC_Program.qpp]: `0.8.13` should read `0.8.13.7` and `obsolete` should read `obsolete / ignored`

File: lib/QC_Program.qpp [hunk 3]
```diff
@@ -792,7 +791,7 @@ while (wh) {
     @param warning_mask An optional warning mask; see @ref warning_constants for values to combine by binary-or; if this argument is 0 or not given then no warnings will be checked or issued and the return value will always be @ref nothing
     @param source An optional source file name for the code being parsed; this is useful if sections of a file are parsed
     @param offset An optional line offset for use with the \a source parameter; this gives the line offset in the file to the code being parsed
-    @param format_label If this argument is @ref Qore::True "True", then the label is formatted as \c "<run-time-loaded: label>"; if @ref Qore::False "False", then it is passed as-is
+    @param format_label obsolete since %Qore 0.8.13
```
Comment (davidnich) [inline lib/QC_Program.qpp]: as above

File: lib/QC_Program.qpp [hunk 4]
```diff
@@ -816,22 +815,21 @@ pgm.parseCommit();
 
     @throw PROGRAM-PARSE-ERROR this exception is thrown if any parse actions are started while the Program object has running threads
 
-    @since %Qore 0.8.7 the \a source, \a offset, and \a format_label arguments were added
+    @since %Qore 0.8.7 the \a source, \a offset, and \a format_label arguments were added. %Qore 0.8.13 the \a format_label is obsolete
```
Comment (davidnich) [inline lib/QC_Program.qpp]: as above

File: lib/ql_misc.qpp [hunk 5]
```diff
@@ -868,7 +868,7 @@ string get_default_encoding() {
     @param warning_mask An optional warning mask; see @ref warning_constants for values to combine by binary-or; if this arguments is 0 or not given then no warnings will be checked or issued and the return value will always be @ref nothing
     @param source An optional source file name for the code being parsed; this is useful if sections of a file are parsed
     @param offset An optional line offset for use with the \a source parameter; this gives the line offset in the file to the code being parsed
-    @param format_label If this argument is @ref Qore::True "True", then the label is formatted as \c "<run-time-loaded: label>"; if @ref Qore::False "False", then it is passed as-is
+    @param format_label %Qore 0.8.13 is obsolete
```
Comment (davidnich) [inline lib/ql_misc.qpp]: as above

File: lib/ql_misc.qpp [hunk 6]
```diff
@@ -881,24 +881,23 @@ parse(code, filename);
     - Qore::Program::parse()
     - Qore::Program::parsePending()
 
-    @since %Qore 0.8.7 the optional \a warning_mask, \a source, \a offset, and \a format_label arguments were added; also label formatting was added;
+    @since %Qore 0.8.7 the optional \a warning_mask, \a source, \a offset, and \a format_label arguments were added; %Qore 0.8.13 the \a format_label is obsolete;
```
Comment (davidnich) [inline lib/ql_misc.qpp]: as above

**Review by davidnich — CHANGES_REQUESTED** [review]

**Comment by tmandys** [comment]
Docs fixed

**Pull request closed without merging**